### PR TITLE
Fixes #172 - Hide Call for Proposals page from UI

### LINKED ISSUE #172: September 7: Closing the Call for Proposals
On Monday at 9pm UTC the call for proposals closes.
The form should close.
Not sure how to execute this or what should be seen on the page.
any ideas @ScottDowne @hannahkane ?
We could just have the Google standard- this form is now closed

Second side to this is to change the URL - so that Space Wranglers and Pathways can still submitt to the form for another two weeks. I think I can do this on my side.

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #172 

@ScottDowne r? :grin:

### CODE REVIEW

File: pages/proposals-extended.jsx [hunk 1]
```diff
@@ -0,0 +1,335 @@
+var React = require('react');
```
Comment (mmmavis) [inline pages/proposals-extended.jsx]: this basically is a duplicate of the current `proposals.jsx`

File: pages/proposals.jsx [hunk 2]
```diff
@@ -211,121 +11,11 @@ var Proposals = React.createClass({
...
-            </InputCombo>
-            <div id="theme-other-error-message" className="error-message">Other cannot be empty.</div>
-            <div id="theme-error-message" className="error-message">At least one theme is required.</div>
-
-            <InputCombo errorMessage="You must agree to our privacy policy." className="checkbox-input" for="privacyPolicy" type="checkbox">
-              I&lsquo;m okay with Mozilla handling my info as explained in this <a href="https://www.mozilla.org/en-US/privacy/">Privacy Policy</a>.
-            </InputCombo>
-            <button className="button" id="submit-button" ref="submitButton" onClick={this.onSubmit}>Submit <Icon spin name="spinner"/></button>
-            <div id="generic-error" className="error-message">Uh oh! There's an unknown error with your session proposal. Please try again later, or email <a href="mailto:festival@mozilla.org ">festival@mozilla.org</a> for help.</div>
+            <h1>Thanks for your interest!</h1>
```
Comment (mmmavis) [inline pages/proposals.jsx]: original suggested copy provided by @hannahkane was "Thanks for your interest in MozFest!" but I removed the last 2 words so we can fit everything into one line

Comment (ScottDowne) [inline pages/proposals.jsx]: This doesn't feel like enough? Shouldn't we mention that proposal deadline has passed?

**Comment by ScottDowne** [comment]
I say land this, and we can update the copy in another ticket. R+

**Comment by hannahkane** [comment]
Copy (if I understand correctly) is here: https://github.com/mozilla/2015.mozillafestival.org/issues/172#issuecomment-136474891

**Comment by mmmavis** [comment]
@hannahkane 

@ScottDowne suggested a 2-line header version
> "Session proposal deadline has passed" 
"Thank you for your interest."

I like it since it's more informative. What do you think?

**Comment by hannahkane** [comment]
That's fine.

**Pull request merged**